### PR TITLE
Support PHP 8.1

### PR DESCRIPTION
**Pull request by GrahamCampbell**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -22,17 +22,48 @@ jobs:
...
+        run: composer config platform.php 8.0.999
+        if: matrix.php > 8
+
+      - name: Download dependencies
+        run: composer update --no-interaction --optimize-autoloader --prefer-stable
+
+      - name: Run tests
+        run: ./vendor/bin/simple-phpunit
+        env:
+          SYMFONY_PHPUNIT_VERSION: 9.5
```
Comment (Nyholm) [inline .github/workflows/ci.yml:59]: Can we let the phpunit bridge decide the version?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -22,17 +22,48 @@ jobs:
...
 
       - name: Download dependencies
-        run: composer update --no-interaction --prefer-dist --optimize-autoloader --prefer-stable
+        run: composer update --no-interaction --optimize-autoloader --prefer-stable
 
       - name: Run tests
         run: ./vendor/bin/simple-phpunit
 
+  build8:
+    name: Build PHP 8
```
Comment (Nyholm) [inline .github/workflows/ci.yml:31]: Why can't we use the "build" job?

**Review by Nyholm — COMMENTED** [review]
Thank you

File: .github/workflows/ci.yml (hunk 1 above)
Comment (GrahamCampbell) [reply .github/workflows/ci.yml:59]: No, because it chooses 9.4 which causes the tests to fail on PHP 8.1.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (GrahamCampbell) [reply .github/workflows/ci.yml:31]: Easier to separate it out to force PHP 9.5.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (GrahamCampbell) [reply .github/workflows/ci.yml:59]: This change just forces it to use the latest PHPUnit 9.x on PHP 8, which is reasonable IMO.

Comment (villfa) [reply .github/workflows/ci.yml:59]: FYI, this will be fixed in the next Symfony releases: https://github.com/symfony/symfony/pull/43200

**Pull request merged**